packages/block-library/src/paragraph/edit.native.js

```diff
@@ -102,6 +118,7 @@ class ParagraphEdit extends Component {
 					}
 					}
 					onSplit={ this.splitBlock }
+					onReplace={ this.onReplace }
```

Comment:
It doesn't seem `onReplace` is used in `rich-text/index.native.js`, should we remove this prop?

Reply:
Right, sorry. It will probably be used later when we implement cut&paste, in beta.

This is ready for another round of testing.